spec/classroom_spec.rb

```diff
@@ -0,0 +1,14 @@
+require_relative '../student'
+require_relative '../classroom'
+
+describe ClassRoom do
+  context 'Check Classroom Assignment ' do
+    it 'should Asign a student to a class' do
+      new_student = Student.new(nil, 12, 'Student1')
+      new_class = ClassRoom.new('CS101')
+      new_class.add_students(new_student)
+      expect(new_class.students[0].name).to eq('Student1')
+      expect(new_student.classroom.label).to eq('CS101')
+    end
```

Comment:
- [ ] Nice work adding the test to the `add_students` method :clap: However it would be nice to test the class `ClassRoom` as well in a different `it` or `context` block
  **Example:**
```ruby
it 'should create a class room' do
  class_room = ClassRoom.new('classroom1')
  expect(class_room.label).to eq('classroom1')
end
```